Jenkinsfile

```diff
@@ -3,9 +3,6 @@
 properties([
     buildDiscarder(logRotator(daysToKeepStr: '20', numToKeepStr: '30')),
 
-    [$class: 'GithubProjectProperty',
-     projectUrlStr: 'https://github.com/coreos/mantle'],
```

Comment:
This property is only here because it was carried over from the previous Jenkins implementation, and I don't really care if we remove it, but is this causing problems?  This is only used for cosmetic enhancements such as adding the project side-bar link on the job and some changelog links (as far as I know).